build.gradle.kts

```diff
@@ -210,6 +210,24 @@ if (hasProperty("fin")) {
     }
 }
 
+tasks.create("run-auctionmark", JavaExec::class) {
```

Comment:
I'd likely stick this in the bench module, which I think would save you a little bit of the faffery here and in your bb script around current directories etc?

Reply:
Should I then add a `./gradlew` script to the `modules/bench` directory or just invoke as you describe? That would still be from the root directory then.